src/WebJobs.Script.WebHost/Security/Authentication/SecurityConstants.cs

```diff
@@ -12,5 +12,6 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Security.Authentication
     public class SecurityConstants
     {
         public const string AuthLevelClaimType = "http://schemas.microsoft.com/2017/07/functions/claims/authlevel";
+        public const string AuthLevelKeyNameClaimType = "http://schemas.microsoft.com/2017/07/functions/claims/authlevelkeyname";
```

Comment:
I had used something simpler in https://github.com/Azure/azure-webjobs-sdk-script/pull/1747 like keyId or perhaps keyName?

Reply:
I like that. We can change the scheme for both here as well. I'll change to `keyid`